### PR TITLE
factory: fix appending to caches

### PR DESCRIPTION
**Pull request by mfojtik**

### CODE REVIEW

File: pkg/controller/factory/factory.go [hunk 1]
```diff
@@ -136,33 +136,34 @@ func (f *Factory) ToController(name string, eventRecorder events.Recorder) Contr
...
 
 	c := &baseController{
-		name:        name,
-		sync:        f.sync,
-		resyncEvery: f.resyncInterval,
-		syncContext: ctx,
+		name:         name,
+		sync:         f.sync,
+		resyncEvery:  f.resyncInterval,
+		cachesToSync: f.cachesToSync,
```
Comment (sttts) [inline pkg/controller/factory/factory.go]: `append(nil, f.cachesToSync...)`

Comment (mfojtik) [reply pkg/controller/factory/factory.go]: fixed

**Pull request merged**